File: Bridge/ConcreteImplementation.py
#!/usr/bin/env python3
# -*- encoding: utf-8 -*-
#
# File name     : ConcreteImplementation.py
#

from Implementation import Implementation

class ConcreteImplementation_A(Implementation):
    def operation_implementation(self):
        return "ConcreteImplementation_A : Here is the result on the platform A.\n"
    
class ConcreteImplementation_B(Implementation):
    def operation_implementation(self):
        return "ConcreteImplementation_B : Here is the result on the platform B.\n"

File: Structural_Patterns/Bridge/c#/ConcreteImplementation.cs
/*!
 *  \file       ConcreteImplementation.cs
 *  \brief      
 *  
 */


using System;

using DesignPatterns.Structural_Patterns.Bridge.IImplementation;

namespace DesignPatterns.Structural_Patterns.Bridge
{
    class ConcreteImplementationA : IImplementation
    {
        public string OperationImpl()
        {
            return "ConcreteImplementationA: The result in platform A.\n";
        }
    }

    class ConcreteImplementationB : IImplementation
    {
        public string OperationImpl()
        {
            return "ConcreteImplementationB: The result in platform B.\n";
        }
    }
}